drivers/wifi/nxp/Kconfig.nxp

```diff
@@ -314,43 +314,69 @@ config NXP_WIFI_SHELL
 
 menu "Wi-Fi driver Stack configurations"
 
-config NXP_WIFI_MON_TASK_STACK_SIZE
+config NXP_WIFI_MON_STACK_SIZE
```

Comment:
Why rename these Kconfigs/change defaults? Ping @MaochenWang1 to take a look at this.

Reply:
Don't need to rename these Kconfigs. I have a Zephyr side PR to change the thread stack size and priority:
https://github.com/zephyrproject-rtos/zephyr/pull/81693
The corresponding hal_nxp(wifi_nxp)  change was done for downstream, but not upstream yet.